Code/src/main/java/nl/utwente/group10/ui/CustomUIPane.java

```diff
@@ -27,6 +33,7 @@
 public class CustomUIPane extends TactilePane {
     private ObjectProperty<Optional<Block>> selectedBlock;
     private ConnectionCreationManager connectionCreationManager;
+    private BooleanProperty errorOccurred;
```

Comment:
This could use some Javadoc. What kind of error are we talking about? What part of the code could set this flag? What part would be interested in reading it?